promscale/installation/kubernetes.md

```diff
@@ -73,8 +93,18 @@ can provide the database URI, or specify connection parameters.
     ```bash
     helm repo update
     ```
-1.  Open the `values.yaml` configuration file, download the tobs
-    `values.yaml` from [here][tobs-values-yaml] and locate the `connection`
+1.  Create a database called `tsdb` for Promscale data:
+    ```bash
+    MASTERPOD="$(kubectl get pod -o name --namespace <namespace> -l release=<releaseName>,role=master)"
+    kubectl exec -i --tty --namespace default ${MASTERPOD} -- psql -U postgres
+    CREATE DATABASE tsdb WITH OWNER postgres;
```

Comment:
Should these be made into separate steps?

Reply:
I don't think so @VineethReddy02, correct? However, as shown, I believe we'll need the `\` at the end of each line.

Essentially, the "MASTERPOD" is a variable passed into the next command, and the `psql` command what we're asking the `kubectl exec` to run.

But again, if someone copied and pasted these three separate lines, it wouldn't work without the `\` on the first two lines to signal continuation.

Reply:
That's my understanding as well. @VineethReddy02 please confirm.

Reply:
I have slightly updated these instructions and I have tested them from my end, works fine. These instructions are taken from here in upstream [timescaledb helm docs](https://github.com/timescale/timescaledb-kubernetes/tree/master/charts/timescaledb-single#connecting-from-inside-the-cluster) and they do not need `\` explicitly at the end of each instruction.